api-outbound.yaml

```diff
@@ -120,6 +120,57 @@ paths:
         basicauth:
           username: "${NEXPOSE_API_USERNAME}"
           password: "${NEXPOSE_API_PASSWORD}"
+  /api/3:
+    get:
+      description: Nexpose API root endpoint, used for verifying if Nexpose can be reached
+      responses:
+        200:
+          description: "Success"
+          content:
+            application/json:
```

Comment:
Question: do we need to declare the content if all we care about is the response status code?  If not, can also get rid of `"responsevalidation"` below